dbt_meshify/utilities/linker.py

```diff
@@ -214,12 +216,26 @@ def dependencies(
 
         return dependencies
 
+    def delete_source_properties(
```

Comment:
should this be in another class? and/or what's the reason for this over `ResourceFileEditor.remove_resource`?

Reply:
In this case, `ResourceFileEditor.remove_resource` is performing the Change, whereas this method `delete_source_properties` is generating the change that will be performed. Saying this out loud makes me think that this should be renamed!